File: src/main/java/com/medicalsystem/security/SecurityConstants.java
package com.medicalsystem.security;

public class SecurityConstants {

    public static final String SECRET = "SuperSecretString";
    public static final long EXPIRATION_TIME = 10 * 24 * 60 * 60; // 10 days
    public static final String TOKEN_PREFIX = "Bearer ";
    public static final String HEADER_STRING = "Authorization";
    public static final String REGISTER_URL = "/api/auth/register";
    public static final String LOGIN_URL = "/api/auth/login";

}

File: src/main/kotlin/com/medicalsystem/security/SecurityConstants.kt
package com.medicalsystem.security

import io.jsonwebtoken.SignatureAlgorithm

object SecurityConstants {
    val SECRET = "SuperSecretString"
    val EXPIRATION_TIME = 10 * 24 * 60 * 60 * 1000
    val SIGNATURE_ALGORITHM = SignatureAlgorithm.HS256
    val TOKEN_PREFIX = "Bearer "
    val HEADER_STRING = "Authorization"
    val LOGIN_URL = "api/auth/login"
    val REGISTER_URL = "api/auth/register"
}